internal/constants.go

```diff
@@ -6,5 +6,5 @@ const (
 
 	// JSONSchemaVersion is the current schema version output by the JSON encoder
 	// This is roughly following the "SchemaVer" guidelines for versioning the JSON schema. Please see schema/json/README.md for details on how to increment.
-	JSONSchemaVersion = "3.2.1"
+	JSONSchemaVersion = "3.2.3"
```

Comment:
I looked back at my comment and realized my feedback has a typo (my bad!). I meant `3.2.2` (the next addition value).

Reply:
The best way to correct this is to delete the new schema file, change the constant, and regenerate.